guides/plugin-advisories.md

```diff
@@ -0,0 +1,203 @@
+
+or maybe:
+
+"If an attacker is able to convince a logged-in admin user to follow a link
+(phishing emails can be made to look very convincing), then ..."
+
+Proof of concept:
+
+“If a logged-in administrator user clicks the submit button on this form, a javascript alert will display on {{A SPECIFIC PAGE}} (In a real attack the form can be made to auto-submit using Javascript).”
+
+### Unserialize
+
+Title:
+
+Unserialize vulnerability in [plugin name] allows an attacker to run code on the server
+
+Description:
+
+If there are classes with methods like __destruct() that have certain properties, the attacker can ...
+
+### Mitigations
+
+- “Disable the plugin until a new version is released that fixes this bug”
+
+### CVSS Scores
+
+- Need to be considered in isolation: e.g. if it’s merely an enabler of other bad stuff then only consider the impact on e.g. integrity
+- CSRF/XSS vulns (i.e. get a logged-in admin to click a link) are Access Complexity: medium.
+
+## Reporting an advisory
+
+In order to publish advisories responsibly there are a number of steps which must be followed:
+
+- [ ] Ensure you're happy with the text of the findings and the title (see previous sections)
+- [ ] Ensure that the mitigations section says something like 'Disable the plugin until a new version is released that fixes this bug'
+- [ ] Ensure that the 'Timeline' section includes the date that the vulnerability was discovered
+- [ ] Publish the advisory privately (Publish > Visibility > Edit) - Make sure it doesn't get tweeted!
+- [ ] Publish the associated inspection privately in the same way
+
+Note:
+
+- The format of timeline entries should be `YYYY-MM-DD: Message`, e.g. `2013-08-09: Discovered`
+
+The following activities can be completed in any order:
+
+### Notify the plugin Author
+
+Note:
+
+In extreme cases where the advisory will have a significant impact we may need to request an encrypted email connection for these communications. In all other cases, normal email is fine.
+
+- [ ] On the (privately) published page, under "Text version for reports" click the "Click for a blank email with the right subject" link
+- [ ] Copy the email text from "Text version for reports"
+- [ ] Find contact details for the plugin author
+- [ ] Add a humanising intro and send to the plugin author
+- [ ] Add a note to the timeline, mentioning the method - e.g. 2014-01-02: Reported to vendor by email
+- [ ] At the bottom of the page, set the status to "Reported"
+
+As an example of a humanising intro:
+
+> Hello
+> We’ve found a security vulnerability in Version 2.8.2 of the Advanced Access Manager plugin – please see the details below.
+> After you’ve had a chance to investigate could you please let us know a timeline for a fix – our disclosure policy is linked at the bottom of the report.
+> Best regards
+> Duncan
+
+### Request a CVE
+
+- [ ] Hit the "Request CVE" button under "Report emails"
```

Comment:
There's also this: https://docs.google.com/forms/d/e/1FAIpQLSeiY7ldJAx-fjU6eSnXDaX5TB--L1ujCQpmGAKnqBSJOcBShw/viewform